frontend/src/views/groups/tabs/modal/Toggle.tsx

```diff
@@ -0,0 +1,50 @@
+import { AiFillInfoCircle } from "react-icons/ai";
+
+interface ToggleProps {
+  description?: string;
+  label: string;
+  value: boolean;
+  changeHandler: () => void;
+}
+
+export const Toggle = ({
+  description,
+  label,
+  value,
+  changeHandler,
+}: ToggleProps) => (
+  <div className="flex items-center justify-between relative">
+    { description ?
+    <div className="group relative overflow-visible">
+      <div className="absolute top-5 z-10 items-center opacity-0 group-hover:opacity-100 transition-opacity border-gray-200 rounded-2xl bg-white border-2">
```

Comment:
Since you use `z-10` here, which is also the same value as is used by the PersonSelect list view, the hidden content is shown even when the list view from PersonSelect is shown. By either changing this z value to a lower one, or by changing the one on line 51 in PersonSelect.tsx to a larger one, this gets fixed.